beta-src/src/components/ui/WDCountryTable.tsx

```diff
@@ -0,0 +1,177 @@
+import * as React from "react";
+import {
+  Box,
+  Chip,
+  Table,
+  TableBody,
+  TableCell,
+  TableHead,
+  TableRow,
+  tableCellClasses,
+} from "@mui/material";
+import { styled } from "@mui/material/styles";
+import BetIcon from "./icons/country-table/WDBet";
+import CentersIcon from "./icons/country-table/WDCenters";
+import { CountryTableData } from "../../interfaces";
+import DelaysIcon from "./icons/country-table/WDDelays";
+import Device from "../../enums/Device";
+import IntegerRange from "../../types/IntegerRange";
+import PowerIcon from "./icons/country-table/WDPower";
+import UnitsIcon from "./icons/country-table/WDUnits";
+
+interface WDCountryTableProps {
+  countries: CountryTableData[];
+  device: Device;
+  maxDelays: IntegerRange<0, 5>;
+}
+
+interface Column {
+  align?: "right" | "left" | "center";
+  icon?: React.FC;
+  id: keyof CountryTableData;
+  label: string;
+}
+
+const columns: readonly Column[] = [
+  { id: "power", label: "Power", icon: PowerIcon, align: "left" },
+  { id: "unitQty", label: "Units", icon: UnitsIcon, align: "center" },
+  {
+    id: "centerQty",
+    label: "Centers",
+    icon: CentersIcon,
+    align: "center",
+  },
+  {
+    id: "bet",
+    label: "Bet",
+    icon: BetIcon,
+    align: "center",
+  },
+  {
+    id: "delaysLeft",
+    label: "Delays",
+    icon: DelaysIcon,
+    align: "center",
+  },
+];
+
+const WDCountryTable: React.FC<WDCountryTableProps> = function ({
+  countries,
+  device,
+  maxDelays,
+}): React.ReactElement {
+  const mobileLandscapeLayout =
+    device === Device.MOBILE_LANDSCAPE || device === Device.MOBILE_LG_LANDSCAPE;
+  const WDTableCell = styled(TableCell)(() => {
```

Comment:
i think we should move this outside this component.

also, there's a way you can set the responsive css like so:

const responsivePadding = {
    mobile: 6,
    mobileLandscape: '6 16px',
    mobileLg: 6,
} as const;

Reply:
This looks interesting, I'll give it a try

Reply:
Can't move lines 63-64 outside as they are dependent on the `device` variable which is a prop. I can make it a function but I don't see the benefit of doing that. 

`mobileLandscapeLayout` is also being used to determine whether to show the full power label or the abbreviation.